content/en/blog/2021/simple-vms-part-2/index.md

```diff
@@ -0,0 +1,290 @@
+---
+title: "Part 2 - Implementing an easier way to add virtual machines to Istio service mesh"
+description: How to statically configure envoy to join a mesh.
+publishdate: 2022-01-17
+attribution: "Nick Nellis (Solo.io)"
+keywords: [virtualmachine,Istio,networking,envoy]
+---
+![Virtual Machine](./istio-setup.png)
+
+Part 1 - [An easier way to add virtual machines to Istio service mesh](/blog/2021/simple-vms/)
+
+A lot of Istio users would like to connect their virtual machines to their service mesh but the current process is very cumbersome and hard to automate. This series explains a simpler way to connect outside applications securely to your applications running in a service mesh. If you would like to know more about why this might work for you, check out part 1 here.
+
+## Preface
+
+As explained in [part 1](/blog/2021/simple-vms/), to reduce complexity of the virtual machine initialization we will not be registering it with Istio. Instead we will be issuing a certificate directly to the virtual machine and manually configuring the envoy instance to communicate inside the mesh via the east/west gateway. First we will need to organize our certificates so that the service mesh applications will trust these external workloads.
```

Comment:
```suggestion
As explained in [part 1](/blog/2021/simple-vms/), to reduce the complexity of virtual machine initialization, we will not be registering it with Istio using the currently [documented approach](/docs/setup/install/virtual-machine/#configure-the-virtual-machine). Instead, we will be issuing a certificate directly to the virtual machine and manually configuring the envoy instance running in the virtual machine to communicate to the mesh via the east/west gateway. First, we will need to organize our certificates so that the service mesh applications will trust these external workloads.
```